R/write_captions.R

```diff
@@ -203,13 +481,78 @@ write_captions <- function(dat, # converted model output object
   # biomass_ref_pt_units <- # biomass reference point units
 
   ## spawning_biomass (ssb)
-  # ssb_start_year <- # start year of ssb plot
-  # ssb_end_year <- # end year of ssb plot
-  # ssb_units <- # ssb units (plural)
-  # ssb_min <- # minimum ssb
-  # ssb_max <- # maximum ssb
-  # ssb_ref_pt <- # ssb reference point
-  # ssb_ref_pt_units <- # ssb reference point units
+  # start year of ssb plot
+  ssb_start_year <- dat |>
+  dplyr::filter(
+    label == "spawning_biomass",
```

Comment:
does this work for all examples?

Reply:
It works for two of my example dat files. Is it failing with yours? Or are there specific files that I should make sure to test it with?